openedx/core/djangoapps/user_api/accounts/settings_views.py

```diff
@@ -270,6 +271,13 @@ def _get_extended_profile_fields():
         "profession": _("Profession"),
         "specialty": _("Specialty")
     }
+    request = theming_helpers.get_current_request()
+    extended_profile_fields_translations = configuration_helpers.get_value(
+        'extended_profile_fields_translations',
+        {},
+    )
+    translations = extended_profile_fields_translations.get(request.LANGUAGE_CODE, {})
+    field_labels_map.update(translations)
```

Comment:
```suggestion
    request = theming_helpers.get_current_request()
    
    if request:
        extended_profile_fields_translations = configuration_helpers.get_value(
            'extended_profile_fields_translations',
            {},
        )
        translations = extended_profile_fields_translations.get(request.LANGUAGE_CODE, {})
        field_labels_map.update(translations)
```

I thin this change fixes tests